docs/source/vision/usage_guides/vision_properties.rst

```diff
@@ -1,4 +1,4 @@
-.. _vision__properties_guide:
+.. _vision_properties_guide:
```

Comment:
why?? is was same format as EVERYTHING

Reply:
It wasn't. I changed it in all relevant places.

Reply:
in all places its the same format _tabular/vision/nlp__name_of_file

Reply:
There are MANY places which it's not like that. I changed what I found.